examples/wasm/README.md

```diff
@@ -42,42 +42,13 @@ example body
 ## Build WASM Module
 
 Now you want to make changes to the C++ filter ([envoy_filter_http_wasm_example.cc](envoy_filter_http_wasm_example.cc))
-and regenerate to WASM module ([envoy_filter_http_wasm_example.wasm](envoy_filter_http_wasm_example.wasm)).
+and build the WASM module ([envoy_filter_http_wasm_example.wasm](envoy_filter_http_wasm_example.wasm)).
 
-### Build using Docker image with WASM toolchain
-
-1. Build Docker image with WASM toolchain
-   ```shell
-   docker build -t wasmsdk:v2 -f ../../api/wasm/cpp/Dockerfile-sdk ../../api/wasm/cpp/
-   ```
-2. Build example WASM Module
+1. Build WASM module
    ```shell
-   ./docker_build_cpp_wasm.sh
+   bazel build envoy_filter_http_wasm_example.wasm
```

Comment:
Nit: You should probably use full Bazel path here, i.e.
```
bazel build //examples/wasm:envoy_filter_http_wasm_example.wasm
```

Reply:
Done.